qiskit/assembler/__init__.py

```diff
@@ -24,6 +24,24 @@
    :toctree: ../stubs/
 
    assemble_circuits
+   ===============
```

Comment:
I think these examples would be better going in the docstrings for the actual function rather than at the top module level. So this example would go under the `assemble` function in `assembler.py`